feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/domain/nft/list/NftListInteractor.kt

```diff
@@ -62,23 +78,20 @@ class NftListInteractor(
 
     @OptIn(ExperimentalCoroutinesApi::class)
     fun subscribeNftOwnerChanged(): Flow<Unit> {
-        return nftRepository.getPendingSendTransactionsNftIds()
-            .flatMapLatest(::subscribeNftsOwnerAddresses)
-            .onEach {
-                val blockTimeInMillis = chainStateRepository.expectedBlockTimeInMillis(it.chainId)
-                delay(blockTimeInMillis.toLong())
-                syncNftsList()
-            }.map {}
+        return pendingSendNftTransactionRepository.getPendingSendTransactionsNftIds()
+            .flatMapLatest(::subscribeNftsOwnerAccountId)
+            .onEach { syncNftListFromNextBlock(it.chainId) }
+            .map {}
```

Comment:
Can you please record a short video on how nft list update looks like after a successfull transfer? I am still have some concerns about full sync overhead, would be nice to see it in action

Reply:
There is a full nft reload, but this is necessary for fully up-to-date information. For the user, updating the screen is imperceptible

https://github.com/novasamatech/nova-wallet-android/assets/140508492/7c50ed47-eb66-452b-883f-928c23729da3